lib/ansible/modules/cloud/vultr/vr_firewall_group_facts.py

```diff
@@ -103,6 +103,9 @@ def get_firewall_group(self):
 
 
 def parse_fw_group_list(fwgroups_list):
+    if isinstance(fwgroups_list, list):
```

Comment:
I would prefer to test for falsy:

~~~python
if not fwgroups_list:
    return []
~~~